scripts/services/migrations.ts

```diff
@@ -1,10 +1,9 @@
+// deno-lint-ignore-file
```

Comment:
why?

Reply:
@MattDHill do we want this?  I think your vscode added it for some reason

Reply:
Can delete